### DISCUSSION TITLE
Why is Gollum called Gollum?

### DISCUSSION THREAD

**Discussion by andrerav** [gollum/gollum#1828] [Q&A]
Hi,
I'm just wondering why the name Gollum was chosen for this project?
Tnx bye,
Andreas

**Comment by dometto** [comment]
Honestly, not a clue! It was initially open sourced by github under that name, and that already came with the namespace `Precious` and the LoTR tests/examples. We've just kept it that way :)

**Comment by grabear** [comment]
Thank you for your service 🫡

**Comment by samuelfullerthomas** [comment]
I reckon @mojombo knows?
